rllib/agents/cql/cql_sac_tf_policy.py

```diff
@@ -264,6 +268,10 @@ def cql_loss(policy: Policy, model: ModelV2, dist_class: Type[TFActionDistributi
         # combined in one loss here.
         final_total_loss += alpha_prime_loss
 
+    if vib_regularizer["enable_regularizer"]:
```

Comment:
I think this should be moved to SAC because the VIB Layer is actually managed in SACTFModel.
 If not, you need to move everything to CQLSACTFModel. I prefer the first option, moving to SAC)

Reply:
Great suggestion. Moved to SAC